### ISSUE TITLE
Optional Focus Indicator doesn't work in fullscreen mode.

### ISSUE THREAD

**Issue by duhrer** [fluid-lab/gamepad-navigator#168]
In #162, an optional focus indicator was added.  This uses a layer similar to the modal manager, but unlike the modal manager, the focus indicator is not visible when working with a fullscreen element on a page like [this one](https://player.vimeo.com/video/166157781).

This may be a layering issue, or we may have to look at what we did with the modal manager to make it work in fullscreen mode.

**Comment by duhrer** [comment]
There is no special logic to the modal manager.  My best guess is that because the modal receives focus, it jumps in front of the fullscreen element on its own.  We may need to watch the state of fullscreen and move the focus indicator back and forth.  This may require more of the DOM manipulation used for #155.
